api/peripheral_api/routes/RFID.js

```diff
@@ -0,0 +1,79 @@
+const express = require('express');
+const router = express.Router();
+const RFID = require('../models/RFID');
+const {
+  checkIfTokenSent,
+  checkIfTokenValid,
+} = require('../../util/token-functions');
+const membershipState = require('../../util/constants').MEMBERSHIP_STATE;
+const { OK, BAD_REQUEST, UNAUTHORIZED, FORBIDDEN, NOT_FOUND } =
+  require('../../util/constants').STATUS_CODES;
+const { RfidHelper } = require('../util/RFID-helpers');
+const awsIot = require('aws-iot-device-sdk');
+const { AWS_IOT_ENDPOINT } = require('../../config/config.json');
+
+const rfidHelper = new RfidHelper();
+
+if (rfidHelper.keysExist()) {
+  const device = awsIot.device({
+    keyPath: '../api/config/AWS-IOT/private.pem.key',
+    certPath: '../api/config/AWS-IOT/cert.pem.crt',
+    caPath: '../api/config/AWS-IOT/AmazonRootCA1.pem',
+    clientId: 'CentauriServer',
+    host: AWS_IOT_ENDPOINT
+  });
+
+  device
+    .on('connect', function() {
+      /* eslint-disable-next-line */
+      console.log('Connected to AWS IoT!');
+      device.subscribe('MessageForNode');
```

Comment:
tbh i would remove, we can add logging to the sce backend differently
```suggestion
      device.subscribe('MessageForNode');
```